scripts/index.js

```diff
@@ -0,0 +1,707 @@
+        player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.videoBuffer.buffered);
+
+      // demuxed audio
+      var audioBuffer = getBuffered(player.tech(true).vhs.masterPlaylistController_.audioSegmentLoader_.sourceUpdater_.audioBuffer &&
+        player.tech(true).vhs.masterPlaylistController_.audioSegmentLoader_.sourceUpdater_.audioBuffer.buffered);
+
+      // muxed audio
+      if (!audioBuffer) {
+        audioBuffer = getBuffered(player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.audioBuffer &&
+          player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.audioBuffer.buffered);
+      }
+      audioBufferedStat.textContent = audioBuffer;
+
+      if (player.tech(true).vhs.masterPlaylistController_.audioSegmentLoader_.sourceUpdater_.audioBuffer) {
+        audioTimestampOffset.textContent = player.tech(true).vhs.masterPlaylistController_.audioSegmentLoader_.sourceUpdater_.audioBuffer.timestampOffset;
+
+      } else if (player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.audioBuffer) {
+        audioTimestampOffset.textContent = player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.audioBuffer.timestampOffset;
+      }
+
+      if (player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.videoBuffer) {
+        videoTimestampOffset.textContent = player.tech(true).vhs.masterPlaylistController_.mainSegmentLoader_.sourceUpdater_.videoBuffer.timestampOffset;
+      }
+
+      // bitrates
+      var playlist = player.tech_.vhs.playlists.media();
+
+      if (playlist && playlist.attributes && playlist.attributes.BANDWIDTH) {
+        videoBitrateState.textContent = (playlist.attributes.BANDWIDTH / 1024).toLocaleString(undefined, {
+          maximumFractionDigits: 1
+        }) + ' kbps';
+      }
+      if (player.tech_.vhs.bandwidth) {
+        measuredBitrateStat.textContent = (player.tech_.vhs.bandwidth / 1024).toLocaleString(undefined, {
+          maximumFractionDigits: 1
+        }) + ' kbps';
+      }
+
+    }, 100);
+  };
+
+  [
+    'debug',
+    'autoplay',
+    'muted',
+    'fluid',
+    'minified',
+    'sync-workers',
+    'liveui',
+    'llhls',
+    'url',
+    'type',
+    'keysystems',
+    'buffer-water',
+    'exact-manifest-timings',
+    'pixel-diff-selector',
+    'override-native',
+    'preload',
+    'mirror-source'
+  ].forEach(function(name) {
+    stateEls[name] = document.getElementById(name);
+  });
+
+  window.startDemo = function(cb) {
+    var state = loadState();
+
+    Object.keys(state).forEach(function(elName) {
+      setInputValue(stateEls[elName], state[elName]);
+    });
+
+    Array.prototype.forEach.call(sources.options, function(s, i) {
+      if (s.value === state.url) {
+        sources.selectedIndex = i;
+      }
+    });
+
+    stateEls.fluid.addEventListener('change', function(event) {
```

Comment:
new button/option. Puts the player in fluid mode.
Have a min-height and aspect-ratio so that when the player reloads content doesn't shift around.